lib/WebpackConfig.js

```diff
@@ -657,6 +663,9 @@ class WebpackConfig {
     }
 
     enableForkedTypeScriptTypesChecking(forkedTypeScriptTypesCheckOptionsCallback = () => {}) {
+        if (this.useBabelTypeScriptPreset) {
+            throw new Error('Encore.enableBabelTypeScriptPreset() can not be called when Encore.enableForkedTypeScriptTypesChecking() has been called.');
```

Comment:
Shouldn't that be `Encore.enableForkedTypeScriptTypesChecking() can not be called when Encore.enableBabelTypeScriptPreset() has been called.`?

Not a big difference but since the exception will be thrown on the `enableForkedTypeScriptTypesChecking()` call it makes more sense to say that it's the "wrong" one.

Reply:
Ah yes, it was a real mess to deal with those error messages :D 

It has been fixed, thanks!